packages/grafana-data/src/transformations/fieldReducer.ts

```diff
@@ -271,33 +271,41 @@ export const fieldReducers = new Registry<FieldReducerInfo>(() => [
   },
 ]);
 
+// Used for test cases
+export const defaultCalcs: FieldCalcs = {
+  sum: 0,
+  max: -Number.MAX_VALUE,
+  min: Number.MAX_VALUE,
+  logmin: Number.MAX_VALUE,
+  mean: null,
+  last: null,
+  first: null,
+  lastNotNull: null,
+  firstNotNull: null,
+  count: 0,
+  nonNullCount: 0,
+  allIsNull: true,
+  allIsZero: true,
+  range: null,
+  diff: null,
+  delta: 0,
+  step: Number.MAX_VALUE,
+  diffperc: 0,
+
+  // Just used for calculations -- not exposed as a stat
+  previousDeltaUp: true,
+};
+
 export function doStandardCalcs(field: Field, ignoreNulls: boolean, nullAsZero: boolean): FieldCalcs {
-  const calcs: FieldCalcs = {
-    sum: 0,
-    max: -Number.MAX_VALUE,
-    min: Number.MAX_VALUE,
-    logmin: Number.MAX_VALUE,
-    mean: null,
-    last: null,
-    first: null,
-    lastNotNull: null,
-    firstNotNull: null,
-    count: 0,
-    nonNullCount: 0,
-    allIsNull: true,
-    allIsZero: true,
-    range: null,
-    diff: null,
-    delta: 0,
-    step: Number.MAX_VALUE,
-    diffperc: 0,
-
-    // Just used for calculations -- not exposed as a stat
-    previousDeltaUp: true,
-  };
+  const calcs: FieldCalcs = { ...defaultCalcs };
```

Comment:
this need to avoid being a reference otherwise things break 😬

Reply:
Love when that happens. 😂